### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jrobinAV**
Archive taipy-config

### CODE REVIEW

File: src/taipy/config/config.pyi
Comment (jrobinAV) [inline src/taipy/config/config.pyi:1]: Note that this file is automatically generated. It has been committed automatically.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+Taipy has performed a repository restructuring. The taipy-config code has been moved and  
+merged into the main [taipy repository](https://github.com/Avaiga/taipy).
 
+<br>
 
-# Taipy config
 
-## License
+# License
 Copyright 2023 Avaiga Private Limited
```
Comment (FabienLelaquais) [inline README.md]: So we don't update the date hu? Now would be a good time though

File: src/taipy/config/config.pyi
Comment (FabienLelaquais) [inline src/taipy/config/config.pyi:1]: Changes on this??

**Review by FabienLelaquais — DISMISSED** [review]
👌

File: README.md (hunk 1 above)
Comment (jrobinAV) [reply README.md]: Yes, indeed. Let me handle the change then.

**Review by FabienLelaquais — APPROVED** [review]
👍👍

**Pull request merged**